UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareServiceClient.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace UKHO.ExchangeSetService.Common.Helpers
+{
+    public class FileShareServiceClient : IFileShareServiceClient
+    {
+        private readonly HttpClient httpClient;
+
+        public FileShareServiceClient(HttpClient httpClient)
+        {
+            this.httpClient = httpClient;
+        }
+
+        public async Task<HttpResponseMessage> CallFileShareServiceApi(HttpMethod method, string postBody, string authToken, string uri)
```

Comment:
PUT method will also have request body. Please can we rename it accordingly.
```suggestion
        public async Task<HttpResponseMessage> CallFileShareServiceApi(HttpMethod method, string requestBody, string authToken, string uri)
```

Reply:
Resolved.